src/tools/stringTools.cpp

```diff
@@ -32,6 +32,13 @@
 
 #include <iostream>
 #include <iomanip>
+#include <regex>
+
+#ifdef _WIN32
+    const std::regex reservedCharsReg("[<>:\"/\\\\|?*]");
+#else
+    const std::regex reservedCharsReg("/");
+#endif
```

Comment:
Move this inside `getSlugifiedFileName()` - it is not a function about whose performance we should care.

Reply:
Done